### PR TITLE
Update onboarding-getting-started eligibility language

### PR DESCRIPTION
**Pull request by analoo**
#### 🔗 Jira ticket
This ticket adds eligibility information in the onboarding-getting started page, pet Carl's comment: https://codeforamerica.atlassian.net/browse/CCAP-234?focusedCommentId=11042

#### ✍️ Description
The strings describing eligible activities have been copied over from the FAQ Screen. These differ from Carl's comment in the Jira Ticket and the design but they are matching what is in FAQ because as Karen confirmed: (1) They should match. (2) the FAQ page has language already reviewed by the IDHS partners: https://cfastaff.slack.com/archives/C0648BQM6UX/p1721685187967209?thread_ts=1721682568.423519&cid=C0648BQM6UX

#### 📷 Design reference
image

#### ✅ Completion tasks

- [ ] Added relevant tests
- [ ] Meets acceptance criteria

### CODE REVIEW

File: src/main/resources/templates/gcc/onboarding-getting-started.html [hunk 1]
```diff
@@ -24,7 +24,19 @@
...
+              <p th:text="#{onboarding-getting-started.accordion.second.paragraph-1}"></p>
+              <p th:text="#{onboarding-getting-started.accordion.second.paragraph-2}"></p>
+              <ul class="list--bulleted">
+                <li th:text="#{faq.do-i-qualify-for-ccap.activities.li.working}"></li>
+                <li th:text="#{faq.do-i-qualify-for-ccap.activities.li.enrolling}"></li>
+                <li th:text="#{faq.do-i-qualify-for-ccap.activities.li.experiencing-homelessness}"></li>
+                <li th:text="#{faq.do-i-qualify-for-ccap.activities.li.deployed}"></li>
+                <li th:text="#{faq.do-i-qualify-for-ccap.activities.li.participating}"></li>
+                <li th:text="#{faq.do-i-qualify-for-ccap.activities.li.caring}"></li>
+                <li th:text="#{faq.do-i-qualify-for-ccap.activities.li.when-you-are-working}"></li>
```
Comment (rapicastillo) [inline src/main/resources/templates/gcc/onboarding-getting-started.html:36]: For the last four items, it should show up as: 

- Experiencing homelessness
- Military deployment
- Getting treatment for medical or mental health
- Caring for a foster child or a child that has recently been in foster care

But for me it's showing up as:

- Deployed in the military
- Participating in certain DCFS family support programs (not including Foster Care)
- Caring for a foster child or a child that has recently been in foster care
- When you are working or going to school and the other parent living in the home is getting treatment for medical, physical, or mental health

**Review by rapicastillo — CHANGES_REQUESTED** [review]
Everything looks good except for the copy changes in `messages.properties`

File: src/main/resources/templates/gcc/onboarding-getting-started.html (hunk 1 above)
Comment (analoo) [reply src/main/resources/templates/gcc/onboarding-getting-started.html:36]: @rapicastillo Yes - it should match this page: 
image

**Review by rapicastillo — APPROVED** [review]
👍🏽

**Pull request merged**